swatch-core/src/main/java/org/candlepin/subscriptions/db/model/BillingProvider.java

```diff
@@ -24,17 +24,17 @@
 import jakarta.persistence.Converter;
 import java.util.Map;
 import java.util.Objects;
-import org.candlepin.subscriptions.utilization.api.v1.model.BillingProviderType;
+import org.candlepin.subscriptions.utilization.api.model.BillingProviderType;
 
 /** Billing provider associated with a host. */
 public enum BillingProvider implements StringValueEnum<BillingProviderType> {
-  EMPTY("", BillingProviderType.EMPTY),
+  EMPTY("", BillingProviderType.__EMPTY__),
```

Comment:
Why would we want it `__EMPTY__` instead of EMPTY?

Reply:
This type is now generated by json schema generator instead of openapi, and the enum name changed.